library/systemd_service.py

```diff
@@ -129,6 +129,8 @@ def main():
             if not changed:
                 module.exit_json(changed=False, result="ok")
             else:
+                cmd_stop = 'systemctl stop %s.service' % module.params['name']
+                os.system(cmd_stop)
```

Comment:
We use this module for installing and configuring systemd unit files. I would recommend using the ansible service module if you want to stop/start services.